minidump-processor/Cargo.toml

```diff
@@ -27,13 +27,15 @@ mozilla_cab_symbols = ["breakpad-symbols/mozilla_cab_symbols"]
 async-trait = "0.1.51"
 breakpad-symbols = { version = "0.15.2", path = "../breakpad-symbols" }
 debugid = "0.8.0"
+cachemap = { git = "https://github.com/afranchuk/cachemap" }
```

Comment:
Sadly I have done so a number of times, most recently a week ago. I've gotten no response. The only contact info I could get was the email address on commits, which may not even be real from the looks of it.

I think I'll just have to rename the crate to publish it.

Reply:
It might be better for cachemap to stay where it is (as it is used by other projects of mine) rather than relocating under `rust-minidump`, though I don't feel that strongly about it since it doesn't actually matter much once it's published and fetched from crates.io.